moveit_core/collision_detection_fcl/src/collision_world_fcl.cpp

```diff
@@ -52,6 +52,8 @@
 
 #include <boost/bind.hpp>
 
+rclcpp::Logger logger = rclcpp::get_logger("collision_detection.fcl");
```

Comment:
```suggestion
const rclcpp::Logger LOGGER = rclcpp::get_logger("collision_detection.fcl");
```

Reply:
We have changed the logger name, this should be fixed https://github.com/ros-planning/moveit2/commit/a3011f83f7a76ca3856b5e8b490f587b1af43b7d